cms/djangoapps/contentstore/features/discussion-editor.py

```diff
@@ -18,7 +18,7 @@ def i_see_only_the_settings_and_values(step):
     world.verify_all_setting_entries(
         [
             ['Category', "Week 1", False],
-            ['Display Name', "Discussion", False],
+            ['Display Name', "Discussion", True],
```

Comment:
@sarina 
These two tests fails if I don't change.
And I think this may be because my code gives some xblocks like discussion and text explicit display name, rather than there implicit default display name. So when the patch is applied, the 'Display Name' field is activated by default, while the original code assume it is inactive.
This is the screenshot of original code when running this test just before the call to ```world.verify_all_setting_entries```:
![d_image_before](https://cloud.githubusercontent.com/assets/1275238/6958393/b9f4fcac-d93d-11e4-9070-9f623fd9540f.png)
And this is the screenshot of patch applied when running the test  just before the call to ```world.verify_all_setting_entries```: 
![d_image_before2](https://cloud.githubusercontent.com/assets/1275238/6958394/be267986-d93d-11e4-9e48-570e1171a428.png)